doc/dev/tma.md

```diff
@@ -349,6 +349,11 @@ the TMA domain can be completely inferred from the schedule.
 > We do not have validation on shared memory schedule yet.
 > If you scheduled something invalid, likely you will see misaligned address error or silent wrong result.
 
+> [!WARNING]
```

Comment:
Perhaps there should be a circular buffering section but I added the warning here for now.